composer.json

```diff
@@ -6,7 +6,8 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.0",
-        "doctrine/inflector": "^1.0"
+        "doctrine/inflector": "^1.0",
+        "ramsey/uuid": "^3.8"
```

Comment:
@klaussilveira Let me know if you think we should move this to suggested only instead of requiring it, since using the transformer is optional.